.github/workflows/build-fb-image.yaml

```diff
@@ -37,17 +114,121 @@ jobs:
         with:
           fetch-depth: 0
 
-      - name: Login to Docker Hub
+      - name: Login to GitHub Container Registry
         uses: docker/login-action@v3
         with:
-          username: ${{ secrets.REGISTRY_USER }}
-          password: ${{ secrets.REGISTRY_PASSWORD }}
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Do I need to add a GitHub token to the action's secret?
But that token cannot start with Github, maybe we can change it to GH_TOKEN ?

![image](https://github.com/fluent/fluent-operator/assets/18525465/68196f6b-13e3-46a8-9812-7661d1bc7d27)

Reply:
As far as I know of I am using GITHUB_TOKEN only which will restrict permissions defined in the actions. I referred to the following [document](https://github.blog/changelog/2021-04-20-github-actions-control-permissions-for-github_token/).  I allowed the write permissions globally on a repo, and use this clause in actions

```YAML
permissions:
  contents: read
  packages: write
```

<img width="1234" alt="Allow Permissions on github" src="https://github.com/fluent/fluent-operator/assets/39090092/c136fea4-a492-4e5e-a02f-a61491b70a1e">

Let's check if this works! if not we can create a secret GH_TOKEN and use it.

Reply:
all right, looks like GITHUB_TOKEN is created automatically for each action and I need to create one manually